apiserver/facades/agent/instancemutater/shim.go

```diff
@@ -4,113 +4,14 @@
 package instancemutater
 
 import (
-	"gopkg.in/juju/charm.v6"
-
 	"github.com/juju/juju/core/cache"
-	"github.com/juju/juju/core/lxdprofile"
 	"github.com/juju/juju/state"
 )
 
 type instanceMutaterStateShim struct {
 	*state.State
 }
 
-func (i *instanceMutaterStateShim) Model() (Model, error) {
```

Comment:
Removed: these were left over from when state was used instead of the model cache.